### PR TITLE
Public Catering model

### PR DESCRIPTION
**Pull request by lyssenko**

### CODE REVIEW

File: UtopiaCity/Common/DbInitializer.cs [hunk 1]
```diff
@@ -22,6 +22,7 @@ public static void RegisterSubInitializers()
             subDbInitializers.Add(new ForPassengerInitializer());          
             subDbInitializers.Add(new SportComplexInitializer());
             subDbInitializers.Add(new LifeInitializer());
+	    subDbInitializers.Add(new PublicCateringInitializer());
```
Comment (crocomoth) [inline UtopiaCity/Common/DbInitializer.cs]: not aligned

File: UtopiaCity/Common/Initializers/PublicCateringInitializer.cs [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+            {
+                if (context.Restaurants.Any())
+                {
+                    return;
+                }
+            }
+
+            if (!context.RestaurantTypes.Any())
+            {
+                object[] restaurantType =
```
Comment (crocomoth) [inline UtopiaCity/Common/Initializers/PublicCateringInitializer.cs]: object seems too generic

**Review by crocomoth — APPROVED** [review]

File: UtopiaCity/Common/DbInitializer.cs (hunk 1 above)
Comment (lyssenko) [reply UtopiaCity/Common/DbInitializer.cs]: fix

File: UtopiaCity/Common/Initializers/PublicCateringInitializer.cs (hunk 2 above)
Comment (lyssenko) [reply UtopiaCity/Common/Initializers/PublicCateringInitializer.cs]: fix

**Review by Ibraga96 — APPROVED** [review]

**Review by AnatoliyDubovtsev — APPROVED** [review]

**Pull request closed without merging**